ModulePresentationsForCAP/gap/ModulePresentationsForCAP.gi

```diff
@@ -1700,11 +1700,6 @@ end );
 InstallGlobalFunction( ADD_LIFT_AND_COLIFT_LEFT, 
 
   function( category )
-  local homalg_ring;
-
-  homalg_ring := category!.ring_for_representation_category;
```

Comment:
`homalg_ring` is used as a variable within the implementation of `Lift` below,
so, it mustn't be deleted.